src/main/java/com/wow/server/dto/UserDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.wow.server.dto;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+
+@Data
+public class UserDTO extends UserMinimalDTO {

```

Comment:
Do we want to show this data publicly?
I do not  think we should, some of them are under GDPR

Reply:
@twinzom, can you please comment on that? What is bare minimum we need to send to FE?